solr/contrib/scripting-update-processor/src/java/org/apache/solr/update/processor/StatelessScriptUpdateProcessorFactory.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.solr.update.processor;
+
+/**
+ * @deprecated use {@link ScriptUpdateProcessorFactory} instead.  This class will be removed in an upcoming release.
+ */
+@Deprecated
+public class StatelessScriptUpdateProcessorFactory extends ScriptUpdateProcessorFactory {
```

Comment:
Just an idea, but you could implement the constructor that logs the deprecation.

Reply:
In talking to David, after fighting with the combination of `@deprecated` and no `@Deprecated` and the logging class, we decide to make this a 9.0 breaking change, and not worry about it.  Since it's in a new `scripting` module anyway, under a new path.